src/04_printing.py

```diff
@@ -11,7 +11,7 @@
 # Using the printf operator (%), print the following feeding in the values of x,
 # y, and z:
 # x is 10, y is 2.25, z is "I like turtles!"
-
+print('x is %2d, y is %1.2f, z is %s' % (x, y, z))
 # Use the 'format' string method to print the same thing
-
-# Finally, print the same thing using an f-string
\ No newline at end of file
+print('x is {}, y is {}, z is {}'.format(x, round(y, 2), z))
+# Finally, print the same thing using an f-string
```

Comment:
Looks like you're missing the f-string here; I can see from the other files that you know how to use them though!